03-Bootstrap/assignment/css/style.css

```diff
@@ -0,0 +1,33 @@
+/* index.html, accordion.html, & grid.html: configues the background image */
```

Comment:
Since we're using SCSS it'd be a good idea to use SCSS for generating your css file

Reply:
During office hours, you said I didn't need to use SCSS, just Bootstrap.

Reply:
Well, the idea is to use bootstrap for as much as you can. Since you're writing custom CSS for this it's still best practice to utilize SCSS rather than just plain CSS for this.